resources/todomvc/vanilla-examples/javascript-web-components/src/data.js

```diff
@@ -0,0 +1,17 @@
+export const data = [
```

Comment:
Is this data used anywhere? If you want to keep it for testing or development purpose, can you please add a comment explaining how to use it?

Reply:
removed